src/views/instancetypes/list/UserInstancetypeList.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { FC } from 'react';
+
+import {
+  VirtualMachineInstancetypeModelGroupVersionKind,
+  VirtualMachineInstancetypeModelRef,
+} from '@kubevirt-ui/kubevirt-api/console';
+import { V1beta1VirtualMachineInstancetype } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { ALL_NAMESPACES_SESSION_KEY } from '@kubevirt-utils/hooks/constants';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import usePagination from '@kubevirt-utils/hooks/usePagination/usePagination';
+import { paginationDefaultValues } from '@kubevirt-utils/hooks/usePagination/utils/constants';
+import {
+  ListPageBody,
+  ListPageFilter,
+  useActiveNamespace,
+  useK8sWatchResource,
+  useListPageFilter,
+  VirtualizedTable,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { Pagination } from '@patternfly/react-core';
+
+import UserInstancetypeRow from './components/UserInstancetypeRow';
+import useUserInstancetypeListColumns from './hooks/useUserInstancetypeListColumns';
+
+import '@kubevirt-utils/styles/list-managment-group.scss';
+
+const UserInstancetypeList: FC = () => {
+  const { t } = useKubevirtTranslation();
+  const [activeNamespace] = useActiveNamespace();
+  const [instanceTypes, loaded, loadError] = useK8sWatchResource<
```

Comment:
yes i know, i used it already, but this hook will fetch all except all projects, now I can edit it and add a boolean arg to it but seems redundant and less redable, because It will make branching in the hook, as there isn't any logic in this hook except the ns checking, so when using here directly usek8swatch , is same as using a custom hook, because there is no logic except the fetching.. so i didn't used the hook u recommended and directly fetched.